src/Components/dashboard/dashboardStyles.scss

```diff
@@ -0,0 +1,224 @@
+$font-color: rgb(150, 157, 166);
+$event-container-color: rgb(255, 255, 255);
+
+.event-container {
+  display: flex;
+  flex-direction: column;
+  height: auto;
+}
+
+.user-name {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  width: 100%;
+  height: 100px;
+}
```

Comment:
lets use the power of sass, we can nest selector inside another selectors:
<img width="712" alt="Screen Shot 2022-12-06 at 8 46 25 AM" src="https://user-images.githubusercontent.com/7244087/205928961-895115eb-2ff7-4e37-9f22-1e836d8d807d.png">